tensorflow/python/framework/traceable_stack.py

```diff
@@ -61,8 +61,11 @@ def set_filename_and_line_from_caller(self, offset=0):
     if len(frame_records) >= local_offset:
       # Negative indexing is one-indexed instead of zero-indexed.
       negative_offset = -(local_offset + 1)
-      self.filename, self.lineno = frame_records[negative_offset][:2]
-      return self.SUCCESS
+      if len(frame_records) > local_offset:
```

Comment:
Ah sorry I didn't read this logic carefully enough when I first looked. I think you should change L61 to `if len(frame_records) > local_offset`, and if they're equal fall through to the HEURISTIC_USED case below.

Reply:
@skye I was wondering that too but wasn't sure what falling back to the heuristic entailed. Much cleaner fix, updated.

Reply:
I think it means you might get an internal TF file in error messages when using R, which is probably the best we can do since the R stacktrace isn't available from Python.

Reply:
@skye Makes sense. BTW. Do I need to worry about the build errors? My guess is that they need to be re-triggered?

Reply:
I'm not sure, but I am seeing seemingly unrelated compilation errors... I'll try again.

Reply:
@harshini-gadige do you know what's going on with the failing builds?